README.md

```diff
@@ -18,18 +115,64 @@ $ MLNX_OFED_VERSION=52220 /bin/bash build.sh
 ```
 Now the ovs-docker image created successfully
 
+## Pull docker image from docker hub 
+
+  - Install(https://docs.docker.com/engine/install/centos/) and start docker:
```

Comment:
Change to:
 - `Install and start docker https://docs.docker.com/engine/install/centos/`_
 remove Line 121-123

Reply:
done